hibernate-core/src/main/java/org/hibernate/dialect/Dialect.java

```diff
@@ -588,19 +508,6 @@ public int resolveSqlTypeLength(
 		}
 	}
 
-	/**
-	 * Useful conversion for databases which represent the
-	 * precision of a float(p) using p expressed in decimal
-	 * digits instead of the usual (standard) binary digits.
-	 */
-	protected static Size binaryToDecimalPrecision(int code, Size size) {
-		return code == Types.FLOAT
-				&& size != null
-				&& size.getPrecision() != null
-				? Size.precision( (int) Math.ceil( size.getPrecision() / LOG_BASE2OF10 ) )
-				: size;
-	}
```

Comment:
@beikov why is this method no longer needed?

Reply:
Moved to `BinaryFloatSqlType`

Reply:
Ah, OK, fine, np. The patch is so big I couldn't find it.